### PR TITLE
Blog post & Quick Start content updates

### PR DESCRIPTION
**Pull request by typaulhus**
**What does this PR do?**
- Updates the Announcing Secretless blog post with a few small content updates
- Changes the Quickstart language a bit so it's clear that users will go through a demo, not get Secretless up & running for their environments.

### CODE REVIEW

File: docs/docs/get_started.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ description: Secretless Broker Documentation
 permalink: docs/get_started.html
 ---
 
-To get started trying out the Secretless Broker, you can run through our [quick start demos](/docs/get_started/quick_start.html)
-that allow you to try it out with PostgreSQL, SSH, or an HTTP web service.
+To get started trying out the Secretless Broker, you can run through our [quick start](/docs/get_started/quick_start.html)
+that allows you to try it out with a simple example using PostgreSQL, SSH, or an HTTP web service.
```
Comment (izgeri) [inline docs/docs/get_started.md]: this sounds awkward - maybe just "that allows you to try some simple examples using"

File: docs/docs/get_started/quick_start.md [hunk 2]
```diff
@@ -6,8 +6,7 @@ description: Secretless Broker Documentation
 permalink: docs/get_started/quick_start.html
 ---
 
-Try out the Secretless Broker brokering a connection to a PostgreSQL database, an SSH connection,
-or a connection to an HTTP service authenticating with basic auth.
+Try a simple example using the Secretless Broker to connect a PostgreSQL database, an SSH connection, or a connection to an HTTP service authenticating with basic auth.
```
Comment (dsbyrne) [inline docs/docs/get_started/quick_start.md]: The wording is a little strange here. What about,

"Try a simple example which uses the Secretless Broker to connect to a PostgreSQL database,  open an SSH connection, or authenticate to an HTTP service protected with basic auth."

File: docs/docs/get_started.md (hunk 1 above)
Comment (typaulhus) [reply docs/docs/get_started.md]: i like it. Will update

File: docs/docs/get_started/quick_start.md (hunk 2 above)
Comment (typaulhus) [reply docs/docs/get_started/quick_start.md]: sounds good!

**Pull request merged**